app/src/androidTest/java/com/android/streetworkapp/ui/park/EventOverviewTest.kt

```diff
@@ -0,0 +1,115 @@
+package com.android.streetworkapp.ui.park
+
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.assertIsNotDisplayed
+import androidx.compose.ui.test.assertTextEquals
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performScrollTo
+import com.android.streetworkapp.model.event.Event
+import com.android.streetworkapp.model.event.EventList
+import com.android.streetworkapp.model.park.Park
+import com.android.streetworkapp.model.parklocation.ParkLocation
+import com.android.streetworkapp.ui.navigation.NavigationActions
+import com.android.streetworkapp.utils.toFormattedString
+import com.google.firebase.Timestamp
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mockito.mock
+
+class EventOverviewTest {
+  private lateinit var park: Park
+  private lateinit var navigationActions: NavigationActions
+  private lateinit var event: Event
+  private lateinit var fullevent: Event
+
+  @get:Rule val composeTestRule = createComposeRule()
+
+  @Before
+  fun setUp() {
+    navigationActions = mock(NavigationActions::class.java)
+    val eventList =
+        EventList(
+            events =
+                listOf(
+                    Event(
+                        "1",
+                        "Group workout",
+                        "A fun group workout session to train new skills! \r\n\r\n" +
+                            "Come and join the fun of training with other motivated street workers while progressing on your figures\r\n" +
+                            "We accept all levels: newcomers welcome\r\n\r\n" +
+                            "see https/street-work-app/thissitedoesnotexist for more details",
+                        5,
+                        10,
+                        Timestamp.now(),
+                        "Malick")))
+
+    event = eventList.events.first()
+    fullevent = event.copy(participants = 10, maxParticipants = 10)
```

Comment:
I agree

Reply:
This was indeed a test i wanted to implement, but since the "join an event" button does not have any click action yet it makes it hard to test if the button's behavior is correct for now.
I will comment it for now for clarity